MobaLiveCD.txt

```diff
@@ -0,0 +1,16 @@
+[Section]
+Name = MobaLiveCD 
+Version = 2.1
+License = GPL
+Description = MobaLiveCD is a freeware that will run your Linux LiveCD on Windows thanks to the excellent emulator called "Qemu".
```

Comment:
`thanks to the excellent "Qemu" emulator.`

Reply:
Both seem equally fine...